### ISSUE TITLE
Looks like you are running V for the first time. V repo not found. Cloning...

### ISSUE THREAD

**Issue by 0x9ef** [vlang/v#818]
**V version:** 0.1.10
**OS:** Win 10 x64

**What did you do?** v test.v -o test.exe

**What did you expect to see?** test.exe

 
**What did you see instead?** Looks like you are running V for the first time.
V repo not found. Cloning...

**Comment by medvednikov** [comment]
Try

`v  -o test.exe test.v`

**Comment by 0x9ef** [comment]
Same :(

**Comment by medvednikov** [comment]
Ok you'll have to build it from source for now:

https://github.com/vlang/v#mingw-w64

**Comment by 0x9ef** [comment]
Thanks. Fixed.

**Comment by darkoverlordofdata** [comment]
Fix not working for me:
```
compiler/v run examples/tetris/tetris.v
Looks like you are running V for the first time.
V repo not found. Cloning...
Re-launch V from v/compiler
```
I run it the way it says, and still get the error

**Comment by medvednikov** [comment]
@darkoverlordofdata try `rm ~/.vlang/VROOT` and rebuild again.

**Comment by darkoverlordofdata** [comment]
that didn't work. But I moved to using the latest release zip, and that did. I'll use those going forward, as I expect them to be more stable. The first release zip I tried  was getting the error:
```
inflating: v                       fchmod (file attributes) error: Operation not permitted
```
so I assumed it to be corrupt and switched to build from source. I'll switch back to the release channel since the latest is working. I assume that will be less frustrating for me, since I'm consuming v rather than developing v.
